__tests__/app.test.js

```diff
@@ -164,4 +164,101 @@ describe('GET /api/reviews/:review_id/comments',() => {
             expect(body).toEqual({msg:"review ID is not found in database"})
         })
     })
+})
+
+describe('POST /api/reviews/:review_id/comments',() => {
+    test("responds with a 201 and the data entered into the database if successful",() => {
+        const newComment = {body:"I freaking love this thing", author: "bainesface"}
```

Comment:
Should be a key of 'username' not 'author' in the body. This turns into author in our database